cross/he853/PLIST

```diff
@@ -0,0 +1,2 @@
+bin:bin/he853
+lib:lib/udev/rules.d/80-he853.rules
```

Comment:
Sorry, I made a mistake when I told you to use `lib`.
https://github.com/SynoCommunity/spksrc/wiki/PLIST-files said to use `rsc` for this kind of file.